app/views/shared/contact_forms/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="modal-header offset-sm-1">
+  <h3 id="contactForm">Contact Access Services</h3>
+</div>
+<div class="modal-body offset-sm-1">
+  <%= form_tag contact_path, method: :post, class: 'contact-form', role: 'form' do %>
+    <div class="row">
+      <%= hidden_field_tag :url, request.referer %>
```

Comment:
This might not work when triggered from JavaScript. You could include this partial: https://github.com/sul-dlss/mylibrary/blob/master/app/views/shared/feedback_forms/_reporting_from.html.erb

or the appropriate markup to have the JavaScript do it for us.

Reply:
I just rendered the existing partial.